src/Mono.Android/Android.Graphics/PathIterator.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Android.Runtime;
+
+namespace Android.Graphics
+{	partial class PathIterator
+	{
+
+#if ANDROID_34
+		// This implements an interface method that should be marked as 'default' but isn't.
```

Comment:
*Might* be useful to link to: https://developer.android.com/reference/java/util/Iterator#remove()